src/styles/globals.css

```diff
@@ -33,6 +33,8 @@
     --ring: 222.2 84% 4.9%;
 
     --radius: 0.5rem;
+
+    --font-sans: "BentonSans";
```

Comment:
I'm slightly unsure if this will actually apply the font. See `~/app/layout.tsx`. We are already applying `Inter` as a font. In order to apply a custom font, we may have to import and use the `.woff2` file that @michellewu13 sent using the `localFonts(...)` function.

See: [Optimizing: Fonts](https://nextjs.org/docs/pages/building-your-application/optimizing/fonts#local-fonts)

Reply:
yeah that makes sense, I don't see where the file is so maybe we can do that during standup on thrusday